### ISSUE TITLE
Docs link near at least major features

### ISSUE THREAD

**Issue by harshsbhat** [unkeyed/unkey#2196]
### Preliminary Checks

- [X] I have reviewed https://unkey.com/docs for existing features that would solve my problem

- [X] I have searched for existing feature requests: https://github.com/unkeyed/unkey/issues

- [X] This issue is not a question, general help request, or anything other than a feature request directly related to Unkey. Please ask questions in our Discord community: https://unkey.com/discord.

### Is your feature request related to a problem? Please describe.

Let's say I am new to semantic caching or need to refer to the documentation for semantic caching while I am on the semantic caching page, that is [/semantic-cache](https://app.unkey.com/semantic-cache). It isn't enjoyable to open the docs in the new tab and then search for the feature I am looking for in the docs. Instead having the topic-specific link on the same page can be very helpful.

![image](https://github.com/user-attachments/assets/6e7a9983-a345-4a0b-a49e-2ee5182cf97e)

### Describe the solution

Need to figure out a way to add topic-specific docs links at least for the 4-5 major features that we have on the dashboard.

**Comment by chronark** [comment]
I don't get it, what would a change look like?

**Comment by harshsbhat** [comment]
I mean having a link on the /semantics-cache page specific to semantic-cache docs can help tbh. Similar for ratelimit and authorization.  As it is a annoying process to open the docs in the new tab and then search for the related features. We can do this for popular and hard to understand features right?

**Comment by chronark** [comment]
ah, so just having a docs link/button somewhere in the top right?
I like that

**Comment by harshsbhat** [comment]
> ah, so just having a docs link/button somewhere in the top right? I like that

yep, specific to the topic link. we will need to figure out how to place it to make sure its not looking too cluttered tho

**Comment by chronark** [comment]
Please hold off on that, we will have some discussions around restructuring the page header next week

**Comment by harshsbhat** [comment]
> Please hold off on that, we will have some discussions around restructuring the page header next week

Yeah, I already have 2-3 issues to work on  😊

**Comment by harshsbhat** [comment]
Hey, @chronark how about  having just a [Read More](https://www.unkey.com/docs/apis/introduction) link somewhere here?

Not a BUTTON but just a link to the relevant docs. Should be helpful for at least three major features i.e. 

`Ratelimiting,` `API keys`, `Semantic cache
`

![image](https://github.com/user-attachments/assets/58b9c14d-259a-4e55-a710-11e3037d8cd5)

 screenshot is edited and not actual changes in code

**Comment by chronark** [comment]
didn't see this, I like that idea

**Comment by harshsbhat** [comment]
Shouldn't this be 50 points too?
